### PR TITLE
test(core): add test for convertAttributeStringValuesToValue

### PR DESCRIPTION
**Pull request by bbogdanov**

### CODE REVIEW

File: packages/core/src/internal/utils/identity.spec.ts [hunk 1]
```diff
@@ -202,6 +203,31 @@ describe('Functional Helper: ', () => {
     });
   });
 
+  fdescribe('convertAttributeStringValuesToValue(): ', () => {
```
Comment (mathisscott) [inline packages/core/src/internal/utils/identity.spec.ts]: I don't think we need `fdescribe` here...

File: packages/core/src/internal/utils/identity.spec.ts [hunk 2]
```diff
@@ -202,6 +203,31 @@ describe('Functional Helper: ', () => {
...
+    it('should turn "true" to true', () => {
+      expect(convertAttributeStringValuesToValue('true')).toEqual(true);
+    });
+    it('should turn "false" to false', () => {
+      expect(convertAttributeStringValuesToValue('false')).toEqual(false);
+    });
+    it('should turn "null" to null', () => {
+      expect(convertAttributeStringValuesToValue('null')).toEqual(null);
+    });
+    it('should turn "null" to null', () => {
```
Comment (mathisscott) [inline packages/core/src/internal/utils/identity.spec.ts]: Duplicated test?

File: packages/core/src/internal/utils/identity.spec.ts [hunk 3]
```diff
@@ -202,6 +203,31 @@ describe('Functional Helper: ', () => {
...
+      expect(convertAttributeStringValuesToValue('null')).toEqual(null);
+    });
+    it('should turn "undefined" to undefined', () => {
+      expect(convertAttributeStringValuesToValue('undefined')).toEqual(undefined);
+    });
+    it('should turn "2" to 2', () => {
+      expect(convertAttributeStringValuesToValue('2')).toEqual(2);
+    });
+
+    it('should turn "2" to 2', () => {
```
Comment (mathisscott) [inline packages/core/src/internal/utils/identity.spec.ts]: Duplicated test?

File: packages/core/src/internal/utils/identity.spec.ts (hunk 1 above)
Comment (mathisscott) [inline packages/core/src/internal/utils/identity.spec.ts]: We aren't testing if it passes a string value as a fallthrough...

**Review by mathisscott — CHANGES_REQUESTED** [review]

File: packages/core/src/internal/utils/identity.spec.ts (hunk 1 above)
Comment (bbogdanov) [reply packages/core/src/internal/utils/identity.spec.ts]: Seems I didn't force push the amended changes. It should be all good now

**Review by mathisscott — APPROVED** [review]

**Pull request merged**